### DISCUSSION TITLE
Is there  any  way to compile the python script as an independent plug-in  for AutoCAD

### DISCUSSION THREAD

**Discussion by HOIY14** [chuongmep/CadPythonShell#14] [Q&A]
Is there  any  way to compile the python script as an independent plugs  for AutoCAD.
Like the RhinoScriptCompiler tool. When the   compile done, the plugs can be use as an  independent plug-in.
![微信图片_20240602121019](https://github.com/chuongmep/CadPythonShell/assets/8143737/541c915e-ce3a-4d13-9dbb-5be6959e70bb)

**Comment by chuongmep** [comment]
Hi @HOIY14 , can tell me more detail what you are trying to achive?  May be that one is what you are looking for ? 
Source : https://github.com/mcneel/rhino.inside/tree/master/Autodesk/AutoCAD

**Reply by HOIY14** [reply]
Use CPS as a lightweight development environment. Once written, the python scripts can be released as dynamic libraries(dll) or other forms(.scr) of stand-alone plug-ins.
My goal is to use CPS as an easy and fast functional development environment without using  VS.
PythonShell or VbsShell in Rhino has  a tool to do it( the picture above). Can it be implemented in AutoCAD?

**Reply by chuongmep** [reply]
Hi @HOIY14 , technical it is posible, but I don't promise it will be implement, my time now is very limit and I will achive this repo soon , please contribute if you want improvement for this project . Thank you.
